### ISSUE TITLE
Release on conda-forge

### ISSUE THREAD

**Issue by Luthaf** [metatensor/metatensor#402]
In addition to PyPI, we should also provide the release on [conda forge](https://conda-forge.org/)! 

I think we should have the following packages over there:

- metatensor: empty package that depends on most/all of the other packages below
- metatensor-core
- metatensor-core-python
- metatensor-operations-python
- metatensor-torch
- metatensor-torch-python

Ideally, `metatensor-core-python` and `metatensor-torch-python` would not have their own copy of the shared libraries from `metatensor-core` and `metatensor-torch` but use the shared libraries for the other packages. That requires some changes to the code, so we might leave it to a future version.

**Comment by PicoCentauri** [comment]
Yes, as I said I can help with that. You mentioned a repo that we can use as reference for the rust building?

**Comment by Luthaf** [comment]
We should not have to build the Rust code directly, `metatensor-core` should be built by CMake. The only point will be to install a rust compiler when building the code, but that should be doable.

I might have mentioned chemfiles where I do the separation between a pure C++ package, and then a Python package which depends on the first one and uses the shared library from the pure C++ package.

**Comment by Luthaf** [comment]
libmetatensor-torch is now on conda-forge: https://github.com/conda-forge/libmetatensor-torch-feedstock

**Comment by Luthaf** [comment]
The python packages are all on conda-forge:

- https://github.com/conda-forge/python-metatensor-core-feedstock/
- https://github.com/conda-forge/python-metatensor-torch-feedstock/
- https://github.com/conda-forge/python-metatensor-operations-feedstock/
- https://github.com/conda-forge/python-metatensor-learn-feedstock/
